packages/shared/src/components/modals/ShareModal.spec.tsx

```diff
@@ -112,8 +109,7 @@ describe('ShareModal Test Suite:', () => {
     renderComponent(true, true);
     const btn = await screen.findByTestId(`social-share-@${squads[0].handle}`);
     btn.click();
-    const modal = client.getQueryData(MODAL_KEY);
-    expect(modal.type).toEqual(LazyModal.CreateSharedPost);
```

Comment:
Why we're not checking about these anymore?

Reply:
We are mocking it differently as shown directly below this since as mentioned, the CreateSharedPost is now rendered directly.